Ref bug sh

AlChudi opened the pull request:
Обработка ошибок, создание установочных скриптов
- Исправлена ошибка при доступе к атрибуту 'shape' у NoneType объектов. 
- Заменен метод детекции model.track на model.predict для улучшения качества обнаружения машин. 
- Добавлены установочные скрипты install.sh и install.bat для упрощения настройки окружения.

AlChudi committed:
Создание установочного скрипта

Добавлены установочные скрипты install.sh и install.bat для упрощения настройки окружения

AlChudi committed:
Обработка ошибок

DanilaAkh committed:
Update install.sh

DanilaAkh reviewed (approved):
Добавление комментариев в установочные скрипты

kurdt23 committed:
feat: Добавление комментариев в  установочные скрипты

- Для упрощения понимания работы установочных скриптов и подключению к виртуальному окружению, в случае возникновения ошибок.
- В случае обнаружения ошибки, скрипты дадут обратный ответ

kurdt23 reviewed (approved):
feat: Добавление комментариев в  установочные скрипты
